### PR TITLE
fix batchnorm bug

### PR DESCRIPTION
**Pull request by yzy5630**
fix bug in batch norm in textCNN

### CODE REVIEW

**Comment by yzy5630** [comment]
@brightmart

**Comment by brightmart** [comment]
great job!
